cypress/unit/store-getter-getHarmonizedPreview.cy.js

```diff
@@ -0,0 +1,185 @@
+import { getters } from "~/store";
+
+let store;
+let originalValue;
+
+describe("getHarmonizedPreview", () => {
+
+    beforeEach(() => {
+
+        store = {
+
+            state: {
+
+                appSetting: {
+
+                    missingValueLabel: "missing value"
+                },
+
+                dataDictionary: {
+
+                    annotated: {
+
+                        column1: {
+
+                            transformationHeuristic: ""
+                        }
+                    }
+                }
+
+            }
+        };
+    });
+
+    it.only("float transformation", () => {
+
+        console.log("Pre-setup");
```

Comment:
Are the `console.log` statements left over from debugging?

Reply:
Thought I removed all of those. Will delete and commit.